packages/shared/src/components/cards/ActionButtons.tsx

```diff
@@ -51,12 +50,11 @@ type LastActionButtonProps = {
   onShare?: (post: Post) => unknown;
   onShareClick?: (event: React.MouseEvent, post: Post) => unknown;
   post: Post;
-  copyLinkFeature?: boolean;
 };
 function LastActionButton(props: LastActionButtonProps) {
-  const { onShareClick, post, copyLinkFeature } = props;
+  const { onShareClick, post } = props;
```

Comment:
Should we be updating this function? I think it is a bit confusing to say `onShareClick` when what it does is copy the link.